source/agora/common/crypto/ECC.d

```diff
@@ -425,3 +431,12 @@ unittest
     assert(points[0] == Point.fromString(
             "0x37e8a197247dd01cc27c178dc0465ce826b4f6e312f3ee4c1df0623ef38c51c5"));
 }
+
+unittest
+{
+    // import std.stdio : writefln;
+    // import std.conv : to;
+    // writefln("point= %s", to!string(Scalar.random().toPoint()));
+    auto p = Point.fromString("0xab4f6f6e85b8d0d38f5d5798a4bdc4dd444c8909c8a5389d3bb209a18610511b");
+    assert(p.isValid());
```

Comment:
I wasn't expecting this to fail. Do we have a bug?

Reply:
I wish you saved the scalar too. Do you happen to have it?

Reply:
No but it can be reproduced.